src/VisualStudio/Core/Def/Implementation/PullMemberUp/MainDialog/PullMemberUpDialog.xaml

```diff
@@ -0,0 +1,189 @@
+﻿<vs:DialogWindow
```

Comment:
(I accidentally reviewed parts of each dialog, but think I got the main ideas across that can be applied to both.)